lib/rules/indent-function-args.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * @fileoverview Check for function args proper indentation.
+ * @author uberVU
+ * @copyright 2015 uberVU. All rights reserved.
+ * See LICENSE file in root directory for full license.
+ */
+'use strict';
+var getNodeIndent = require('../get-node-indent.js');
+var indentSize = 4;
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = function(context) {
+
+  var allignedWithParen = {
```

Comment:
Ok but we still have a problem. ESLint will then cry :crying_cat_face: http://eslint.org/docs/rules/no-unused-vars